tests/dummy/app/templates/docs/standalone-apps.md

```diff
@@ -21,6 +21,8 @@ In my case, I did it as an [in repo addon](https://github.com/ember-animation/em
 
 ## 4. Add `ember-cli-addon-docs` to your docs app.
 
+You can follow the <DocsLink @route="quickstart">quickstart guide.</DocsLink>
```

Comment:
@MichalBryxi looks like this didn't work actually, see https://ember-learn.github.io/ember-cli-addon-docs/versions/main/docs/standalone-apps

Reply:
Ah! Mea culpa. Fix: https://github.com/ember-learn/ember-cli-addon-docs/pull/1681